lib/portage/sync/modules/git/git.py

```diff
@@ -370,6 +370,8 @@ def update(self):
         if quiet:
             merge_cmd.append("--quiet")
 
+        writemsg_level(' '.join(merge_cmd) + "\n")
```

Comment:
Maybe only if not quiet? What do you think?

Reply:
Sounds sensible. Should do the same for the "git fetch" message above?

Reply:
I think so yeah